.env.sample

```diff
@@ -25,7 +25,7 @@ DB_NAME='bhima_test'
 SESS_SECRET='XopEn BlowFISH'
 
 # define logging level for the npm debug module.
-DEBUG=app,errors
+DEBUG=app,errors,mailer
```

Comment:
Did you want this in the production version?
BTW, the juicy email messages occur with DEBUG set to ...,bhima:mailer

Reply:
Whoops.  Not sure how that happened.